code/api/src/setup/models.js

```diff
@@ -8,7 +8,8 @@ const models = {
   User: databaseConnection.import('../modules/user/model'),
   Product: databaseConnection.import('../modules/product/model'),
   Crate: databaseConnection.import('../modules/crate/model'),
-  Subscription: databaseConnection.import('../modules/subscription/model')
+  Subscription: databaseConnection.import('../modules/subscription/model'),
+  UserProduct: databaseConnection.import('../modules/userproduct/model')
```

Comment:
Good call on in-general cleaning this all up from how I had abstracted everything.